### DISCUSSION TITLE
shadcn@latest init in remix Generates Incorrect fontFamily.sans Configuration in tailwind.config.ts

### DISCUSSION THREAD

**Discussion by krabhi1** [tailwindlabs/tailwindcss#14949] [Help]
**What version of Tailwind CSS are you using?**

For example: v3.4.4

**What build tool (or framework if it abstracts the build tool) are you using?**

For example: remix v2.14.0

**What version of Node.js are you using?**

For example: v18.20.4

**What browser are you using?**

For example: Brave

**What operating system are you using?**

For example: manjaro linux

**Reproduction URL**

This issue can be reproduced by following the steps to initialize a Remix project with shadcn@latest init, so no specific URL is required.

**Describe your issue**

When using `shadcn@latest init` in a Remix project, the generated _tailwind.config.ts_ file includes a misconfigured fontFamily.sans setting. The issue arises due to newline (\n) characters being embedded within the array values, which causes an error when Tailwind attempts to process the configuration.

![image](https://github.com/user-attachments/assets/f662f8e4-1772-47f5-9076-b470cdeca716)

**Comment by philipp-spiess** [comment]
@krabhi1 Hey! The screenshot indicates that the file that is created by the `shadcn@latest init` script is malformed. I suggest reaching out to the library maintainers with a reproduction to get this resolved.

I did try to run the scripts on a new Remix project though and wasn't able to reproduce the issue. The default script does not seem to add a `fontFamily` overwrite and even if I have a `fontFamily` in the config prior to running the script, the newlines would properly be printed.
